compiler/llvm/llvmGlobalToWide.cpp

```diff
@@ -664,7 +666,7 @@ namespace {
 #else
             Value* call = CallInst::Create(getFn, args, "", oldLoad);
 #endif
-            assert(call);
+            if (call == nullptr) assert(false && "failure creating call");
```

Comment:
Why not just have these be `INT_ASSERT`?

Reply:
AFAIK it can't because this is an LLVM pass and not aware of Chapel compiler stuff at all. Or at least it was designed to be that way. We could arguably make it aware of Chapel compiler stuff but IMO it is better for our custom LLVM passes to be able to be compiled separately etc.

Reply:
Ok, that seems reasonable to me.